kodiak/main.py

```diff
@@ -117,18 +122,17 @@ def find_branch_names_latest(sha: str, branches: List[Branch]) -> List[str]:
 
 
 @webhook()
-async def status_event(status_event: events.StatusEvent) -> None:
+async def status_event(status_event: StatusEvent) -> None:
     """
     Trigger evaluation of all PRs associated with the status event commit SHA.
     """
-    assert status_event.installation
     owner = status_event.repository.owner.login
     repo = status_event.repository.name
     installation_id = str(status_event.installation.id)
     log = logger.bind(owner=owner, repo=repo, install=installation_id)
 
     refs = find_branch_names_latest(
-        sha=status_event.commit.sha, branches=status_event.branches
+        sha=status_event.sha, branches=status_event.branches
```

Comment:
There's a `sha` at the root of the payload as well as on the commit. To keep the event simple we now only parse that top level `sha`.
https://developer.github.com/v3/activity/events/types/#statusevent